src/wizard/wizard-directive.js

```diff
@@ -3,22 +3,36 @@
+  *     <div pf-wizardsubstep><!-- content here --></div>
+  *   </div>
+  *   <div pf-wizardstep><!-- additional configuration can be added here with substeps if desired --></div>
+  *   <div pf-wizardstep><!-- review steps and final command here --></div>
+  * </div>
+  * </pre>
   *
   * @param {string} title The wizard title displayed in the header
-  * @param {boolean} hideIndicators  Hides the step indicators in the header of the wizard
-  * @param {string} currentStep The current step can be changed externally - this is the title of the step to switch the wizard to
-  * @param {string} cancelTitle The text to display on the cancel button
-  * @param {string} backTitle The text to display on the back button
-  * @param {string} nextTitle The text to display on the next button
-  * @param {function (step) } backCallback Called to notify when the back button is clicked
-  * @param {function (step) } nextCallback Called to notify when the next button is clicked
-  * @param {function ()} onFinish Called to notify when when the wizard is complete.  Returns a boolean value to indicate if the finish operation is complete
-  * @param {function ()} onCancel Called when the wizard is canceled, returns a boolean value to indicate if cancel is successful
+  * @param {boolean=} hideIndicators  Hides the step indicators in the header of the wizard
+  * @param {string=} currentStep The current step can be changed externally - this is the title of the step to switch the wizard to
+  * @param {string=} cancelTitle The text to display on the cancel button
+  * @param {string=} backTitle The text to display on the back button
+  * @param {string=} nextTitle The text to display on the next button
+  * @param {function (step)=} backCallback Called to notify when the back button is clicked
+  * @param {function (step)=} nextCallback Called to notify when the next button is clicked
+  * @param {function ()=} onFinish Called to notify when when the wizard is complete.  Returns a boolean value to indicate if the finish operation is complete
+  * @param {function ()=} onCancel Called when the wizard is canceled, returns a boolean value to indicate if cancel is successful
   * @param {boolean} wizardReady Value that is set when the wizard is ready
-  * @param {boolean} wizardDone  Value that is set when the wizard is done
+  * @param {boolean=} wizardDone  Value that is set when the wizard is done
   * @param {string} loadingWizardTitle The text displayed when the wizard is loading
-  * @param {string} loadingSecondaryInformation Secondary descriptive information to display when the wizard is loading
+  * @param {string=} loadingSecondaryInformation Secondary descriptive information to display when the wizard is loading
+  * @param {number=} minHeight The minimum height the wizard should adjust to if the window height is decreased.  The wizard height is adjusted with the window resize event and this sets the minimum.
```

Comment:
@jeff-phillips-18  - I did it this way to keep it inline with the design and js from PatternFly which keeps the height at a minHeight or within 70 pixels of the page.  We'd probably have to go back to the designer to adjust it at this point or we'd be inconsistent between the two.